module/item-sheet.js

```diff
@@ -53,10 +53,10 @@ export class CypherItemSheet extends ItemSheet {
 
     totalModifier = skillRating + modifiedBy;
 
-    if (totalModifier == 1) totalModified = "eased";
-    if (totalModifier >= 2) totalModified = "eased by " + totalModifier + " steps";
-    if (totalModifier == -1) totalModified = "hindered";
-    if (totalModifier <= -2) totalModified = "hindered by " + Math.abs(totalModifier) + " steps";
+    if (totalModifier == 1) totalModified = game.i18n.localize("CYPHERSYSTEM.Eased").toLowerCase();
+    if (totalModifier >= 2) totalModified = game.i18n.localize("CYPHERSYSTEM.EasedBy") + " " + totalModifier + " " + game.i18n.localize("CYPHERSYSTEM.Steps");
+    if (totalModifier == -1) totalModified = game.i18n.localize("CYPHERSYSTEM.Hindered").toLowerCase();
+    if (totalModifier <= -2) totalModified = game.i18n.localize("CYPHERSYSTEM.HinderedBy") + " " + Math.abs(totalModifier) + " " + game.i18n.localize("CYPHERSYSTEM.Steps");
```

Comment:
Needs a different solution. German needs a different sentence structure.

Reply:
This may works with a nested structure, I need to try it

Reply:
Refactored translation